packages/libs/components/src/plots/VolcanoPlot.css

```diff
@@ -0,0 +1,14 @@
+.VolcanoPlotTooltip > ul {
+  margin: 0;
+  padding: 0.25em;
+  list-style: none;
+  line-height: 1.5em;
+}
+
+.VolcanoPlotTooltip > ul > li {
+  font-weight: normal;
+}
```

Comment:
NOTE: I'm just glancing back over this and can't see why this would be necessary. Perhaps I'm misremembering something. If not, I'll remove when I address feedback!